packages/terra-clinical-item-display/src/terra-dev-site/test/clinical-item-display/display/IconItemDisplay.test.jsx

```diff
@@ -3,4 +3,14 @@ import IconSuccess from 'terra-icon/lib/icon/IconSuccess';
 
 import ItemDisplay from '../../../../ItemDisplay';
 
-export default () => <ItemDisplay text="Test Text" icon={<IconSuccess />} id="ItemDisplay" />;
+const testText = 'Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text Test Text';
+
+export default () => (
+  <React.Fragment>
+    <ItemDisplay text={`Center / default - ${testText}`} icon={<IconSuccess />} id="test-icon-center" />
+    <br />
+    <ItemDisplay text={`Top - ${testText}`} icon={<IconSuccess />} id="test-icon-top" iconAlignment="top" />
+    <br />
+    <ItemDisplay text={`Inline - ${testText}`} icon={<IconSuccess />} id="test-icon-inline" iconAlignment="inline" />
```

Comment:
I think the second line wrap of this item is getting cut off in the screenshots below.

Reply:
Good call - inline block needed to be bound in block element to correctly capture edges of screeenshot. Fixed with https://github.com/cerner/terra-clinical/pull/490/commits/720eeeba8d785252384733168dcdb59c0a554097#diff-015ab00051023df5defba34800b2a700